core/sr-primitives/src/generic/unchecked_extrinsic.rs

```diff
@@ -312,63 +310,6 @@ mod tests {
 		);
 	}
 
-	#[test]
-	fn mortal_signed_check_should_work() {
```

Comment:
without the mortality logic here these tests are pointless.

Reply:
then they should be replaced with tests for the `CheckEra` type directly.

Reply:
Yeah will add some unit tests to `system` for it. Also did the same for all other `SignedExtensions`: each module is now responsible to test it properly.

Reply:
added